ci-operator/step-registry/openshift-qe/installer/bm/prereqs/openshift-qe-installer-bm-prereqs-commands.sh

```
@@ -83,14 +99,19 @@ if [[ "$PRE_UEFI" == "true" ]]; then
 EOF
-envsubst '${FOREMAN_OS},${LAB},${LAB_CLOUD},${NUM_WORKER_NODES},${PRE_PXE_LOADER},${PRE_UEFI},${QUADS_INSTANCE},${TYPE}' < /tmp/prereqs.sh > /tmp/prereqs-updated.sh
+envsubst '${FOREMAN_OS},${LAB},${LAB_CLOUD},${NUM_WORKER_NODES},${OCP_INVENTORY_OVERRIDE},${OCPINV},${PRE_PXE_LOADER},${PRE_UEFI},${QUADS_INSTANCE},${TYPE}' < /tmp/prereqs.sh > /tmp/prereqs-updated.sh
 
 # Generate the foreman_config.yml file
 if [[ "$PRE_PXE_LOADER" == "true" ]]; then
   FOREMAN_INSTANCE=$(cat ${CLUSTER_PROFILE_DIR}/foreman_instance_${LAB})
   export FOREMAN_INSTANCE
-  OCPINV=$QUADS_INSTANCE/instack/$LAB_CLOUD\_ocpinventory.json
-  PSWD=$(curl -sSk $OCPINV  | jq -r ".nodes[0].pm_password")
+  if [[ "${OCP_INVENTORY_OVERRIDE}" == "true" ]]; then
+    OCPINV=$(cat ${CLUSTER_PROFILE_DIR}/ocp_inventory_path)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)